diff --git a/node_modules/socket.io/lib/index.js b/node_modules/socket.io/lib/index.js
index 5287e4e..2f4330d 100644
--- a/node_modules/socket.io/lib/index.js
+++ b/node_modules/socket.io/lib/index.js
@@ -112,14 +112,14 @@ Server.prototype.serveClient = function(v){
     }
     return require.resolve(file);
   };
-  if (v && !clientSource) {
-    clientSource = read(resolvePath( 'socket.io-client/dist/socket.io.js'), 'utf-8');
-    try {
-      clientSourceMap = read(resolvePath( 'socket.io-client/dist/socket.io.js.map'), 'utf-8');
-    } catch(err) {
-      debug('could not load sourcemap file');
-    }
-  }
+//  if (v && !clientSource) {
+//    clientSource = read(resolvePath( 'socket.io-client/dist/socket.io.js'), 'utf-8');
+//    try {
+//      clientSourceMap = read(resolvePath( 'socket.io-client/dist/socket.io.js.map'), 'utf-8');
+//    } catch(err) {
+//      debug('could not load sourcemap file');
+//    }
+//  }
   return this;
 };
 
