diff --git a/node_modules/sql-query-identifier/lib/tokenizer.js b/node_modules/sql-query-identifier/lib/tokenizer.js
index f8980fe..bb03059 100644
--- a/node_modules/sql-query-identifier/lib/tokenizer.js
+++ b/node_modules/sql-query-identifier/lib/tokenizer.js
@@ -249,7 +249,7 @@ function skipWord(state, value) {
     };
 }
 function isWhitespace(ch) {
-    return ch === ' ' || ch === '\t' || ch === '\n';
+    return ch === ' ' || ch === '\t' || ch === '\n' || ch == '\r';
 }
 function isString(ch, dialect) {
     const stringStart = dialect === 'mysql' ? ["'", '"'] : ["'"];
