diff --git a/node_modules/dmg-builder/out/dmg.js b/node_modules/dmg-builder/out/dmg.js
index 2120f49..e4a783a 100644
--- a/node_modules/dmg-builder/out/dmg.js
+++ b/node_modules/dmg-builder/out/dmg.js
@@ -297,15 +297,17 @@ async function createStageDmg(tempDmg, appPath, volumeName) {
   //noinspection SpellCheckingInspection
   const imageArgs = addLogLevel(["create", "-srcfolder", appPath, "-volname", volumeName, "-anyowners", "-nospotlight", "-format", "UDRW"]);
 
-  if (_builderUtil().log.isDebugEnabled) {
+  // if (_builderUtil().log.isDebugEnabled) {
     imageArgs.push("-debug");
-  }
+  // }
 
   imageArgs.push("-fs", "HFS+", "-fsargs", "-c c=64,a=16,e=16");
   imageArgs.push(tempDmg); // The reason for retrying up to ten times is that hdiutil create in some cases fail to unmount due to "resource busy".
   // https://github.com/electron-userland/electron-builder/issues/5431
 
-  await (0, _builderUtil().retry)(() => (0, _builderUtil().spawn)("hdiutil", imageArgs), 5, 1000);
+  await new Promise(resolve => setTimeout(resolve, 5000));
+
+  await (0, _builderUtil().retry)(() => (0, _builderUtil().spawn)("hdiutil", imageArgs), 15, 3000);
   return tempDmg;
 }
 
